MainWindow.cpp

```diff
@@ -464,10 +464,20 @@ void MainWindow::on_actionOpen_triggered() {
       this, tr("Open Project"), "",
       tr("All supported formats (*.egm *.yyp *.project.gmx *.gm81 *.gmk *.gm6 *.gmd);;GameMaker: Studio 2 Projects "
          "(*.yyp);;GameMaker: Studio Projects (*.project.gmx);;Classic "
-         "GameMaker Files (*.gm81 *.gmk *.gm6 *.gmd);;All Files (*)"));
+         "GameMaker Files (*.gm81 *.gmk *.gm6 *.gmd);;"
+         "Tiled Tileset (*.tsx);;All Files (*)"));
   if (!fileName.isEmpty()) openFile(fileName);
 }
 
+void MainWindow::on_actionSave_triggered() {

```

Comment:
This probably belongs in a separate pull request?

Reply:
Oh yup, I just thought that it might take more time that way so included it here. But shall I make a separate PR for that?

Reply:
Since the change is unrelated to the Tiled stuff, it should be a separate PR, even if it will take a bit more time that way. :-)